src/grt/src/GlobalRouter.tcl

```diff
@@ -255,7 +255,7 @@ proc global_route { args } {
   }
 }
 
-sta::define_cmd_args "repair_antennas" { diode_cell \
+sta::define_cmd_args "repair_antennas" { [diode_cell] \
                                          [-iterations iterations] \
                                          [-ratio_margin ratio_margin]}
```

Comment:
As diode_cell is a positional argument it should come after the flags

Reply:
Got it, didn't realize that's the convention - updated it.